demo/data.js

```diff
@@ -2951,7 +2951,7 @@ export const demoData = {
         C6: "-445248",
```

Comment:
Some functional notes: 

- hover style of simple arrow (without chip) is worst than before IMO ? particularly on white background
- hover style of arrow/chip is set when hovering on the cell, not on the icon. And the rest of the cell isn't clickable
- whole chip should probably be clickable, not just the arrow ?

Reply:
- cannot use chip in "Values in range" ?

Reply:
- font size looks really small in the autocomplete dropdown

Reply:
- cannot change text color of chip item with dark background. Chip text color should not take priority over user-defined color. Not cf colors.

Reply:
- when chip arrow is white on dark background, hover style is barely noticeable